src/weekview.html

```diff
@@ -1,11 +1,11 @@
 <div class="swiper weekview-swiper" #weekViewSwiper>
     <div class="swiper-wrapper slides-container" [dir]="dir">
-        <div class="swiper-slide slide-container weekview-slide-container">
-            <table class="table table-bordered table-fixed weekview-header">
+        <div *ngFor="let sliderIndex of sliderIndexList" class="swiper-slide slide-container weekview-slide-container">
+            <table class="weekview-table weekview-table-bordered weekview-table-fixed weekview-header">
```

Comment:
still add table table-bordered table-fixed, use "weekview-slide-container > table" to customize.